source/elements/oneDAL/source/algorithms/decomposition/pca.rst

```diff
@@ -32,7 +32,31 @@ oneDAL specifies two methods for PCA computation:
 -----------------------
 Covariance-based method
 -----------------------
-[TBD]
+
+This method uses eigendecomposition of the covariance matrix to find the
+principal components of the dataset. The following steps are distinguished:
+
+#. Covariance computation
+#. Eigenvectors and eigenvalues computation
+#. Resulting matrix formation
+
+Covariance matrix computation shall be performed in the following way:
+
+#. Compute the sums :math:`s_i = \sum_{j=1}^n x_{i,j}, \quad 1 \leq i \leq p`.
+
+#. Compute the cross-product :math:`P = XX^T - s^Ts`.
+
+#. Compute the covariance matrix :math:`\Sigma = \frac{1}{n - 1} P`.
+
+To compute eigenvalues :math:`\lambda_i` and eigenvectors :math:`\upsilon_i`,
+where :math:`1 \leq i \leq p`, the following equation shall be resolved:
+
+.. math::
+   \Sigma\lambda =  v\lambda
+
+The final step is to sort the set of pairs :math:`(\lambda_i, \upsilon_i)` in
```

Comment:
The result consists of several elements - means, variances, etc. The description should reflect it

Reply:
fixed